UndertaleModLib/Models/UndertaleGeneralInfo.cs

```diff
@@ -205,6 +205,21 @@ public enum FunctionClassification : ulong
     /// </summary>
     public UndertaleString Name { get; set; }
 
+    /// <summary>
+    /// Different GameMaker release branches. LTS has some but not all features of equivalent newer versions.
+    /// </summary>
+    public enum BranchType
+    {
+        Unknown,
+        LTS2022_0,
+        Post2022_0
+    }
+
+    /// <summary>
+    /// The GameMaker release branch of the data file. May be set to "NonLTS" when features exempted from LTS are detected.
```

Comment:
"NonLTS" doesn'T exist. please adjust documentation (and use cref tags if it makes sense=